app/views/ReportView/components/GenomicSummary/components/KeyAlterations/index.scss

```diff
@@ -0,0 +1,25 @@
+// Shared styling between both versions
+.genomic-summary, .genomic-summary--print {
+  &__alterations {
+    margin: 0 0 32px 0;
+  }
+}
+
+.genomic-summary {
```

Comment:
this violates separation of concern, shouldn't these be on `genomic-summary.scss` instead? If we add these here it'll end up with unintended effects when we go edit the original `genomic-summary.scss`

Reply:
that makes sense I'll update all of them to be key-alterations instead